HISTORY.md

```diff
@@ -455,37 +460,37 @@
 
  * update range-parser and fresh
 
-0.1.4 / 2013-08-11 
+0.1.4 / 2013-08-11
```

Comment:
Oh, my editor removed the trailing white space.  This seems like a good thing, but I can make a separate commit if you think it worth it.